README-translated/README-Hindi.md

```diff
@@ -0,0 +1,127 @@
+
+*द ट्यूरिंग वे एक किताब, एक समुदाय और एक वैश्विक सहयोग है।.*
+
+छात्रों, शोधकर्ताओं, सॉफ्टवेयर इंजीनियरों, परियोजना नेताओं और संस्थापक टीमों सहित सभी हितधारकों को डेटा विज्ञान पुनरुत्पादन में उनकी भूमिकाओं और जिम्मेदारियों को समझने के लिए द ट्यूरिंग वे का उपयोग करने के लिए आमंत्रित किया जाता है।
+आप [ऑनलाइन](https://github.com/alan-turing-institute/the-turing-way) पुस्तक पढ़ सकते हैं, परियोजना में योगदान देने के लिए हमारे [गाइड](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) में वर्णित परियोजना में योगदान दे सकते हैं और सभी सामग्रियों का पुन: उपयोग करते हैं ([लाइसेंस देखें](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md))।
+
+[![ट्यूरिंग तरीके की ऑनलाइन पुस्तक का स्क्रीनशॉट। यह पुस्तक की शुरुआत में द ट्यूरिंग वे दृष्टांतचित्रों में से एक को भी दिखाता है। इस दृष्टांत में, हम विभिन्न डेटा विज्ञान कौशल के लिए दुकानों के साथ एक सड़क या सड़कें देख सकते हैं। जिन दुकानों में लोग अपनी खरीदारी सूची लेकर प्रवेश कर सकते हैं, उन्हें क्या चाहिए वो लेकर बाहर जा सकते हैं।](../book/website/figures/README_imgs/README_book.png)](https://the-turing-way.netlify.app/welcome.html)
+
+*द ट्यूरिंग वे ऑनलाइन पुस्तक से स्क्रीनशॉट ([इस छवि का उपयोग एक प्रस्तुति में करें](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*
+
+द ट्यूरिंग वे 2019 में एक छोटी डेटा विज्ञान मार्गदर्शिका के रूप में शुरू हुआ, और तब से यह तब तक विस्तार हुआ है और आब यह कई विषयोंको (जैसे की: पुनरुत्पादित अनुसंधान, परियोजना डिजाइन, संचार, सहयोग और नैतिक शोध) कवर करने वाले कई गाइडों द्वारा गठित एक पूर्ण मैनुअल बन गया है। प्रत्येक मार्गदर्शिका विभिन्न विषयों पर अध्याय प्रस्तुत करती है जो अच्छे अभ्यासों और अनुशंसाओं को कवर करते हैं। इन अध्यायों को सहयोगियों द्वारा सह-लिखा गया है जो छात्र, शोधकर्ता, शिक्षक, समुदाय के नेता, राजनेता, और विविध पृष्ठभूमि के पेशेवर जिनको अपने पेशे का जीवित अनुभव और समुचा ज्ञान हैं।
+
+***हमारा लक्ष्य प्रतिलिपि प्रस्तुत करने योग्यता को "ऐसा करना बहुत आसान नहीं है" बनाना है।***
+
+**विषयसूची:**
+
+- [परियोजना के बारे में](#परियोजना-के-बारे-में)
+- [टीम](#टीम)
+- [योगदान](#योगदान)
+- [उद्धरण _द ट्यूरिंग वे_](#उद्धरण-द-ट्यूरिंग-वे)
+- [संपर्क करें](#संपर्क-करें)
+- [योगदानकर्ता](#योगदानकर्ता)
+
+🎧 यदि आप परियोजना के लिए एक ऑडियो परिचय पसंद करते हैं, तो हमारी टीम के सदस्य राचेल ने पोर्टो में [ओपन साइंस फेयर 2019](https://www.opensciencefair.eu/) में प्रस्तुत किया और उसका डेमो [ओरियन पॉडकास्ट](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/) द्वारा रिकॉर्ड किया गया था। _द ट्यूरिंग वे_ सिंहावलोकन मिनट 5:13 से शुरू होता है।
```

Comment:
```suggestion
🎧 यदि आप परियोजना के लिए एक ऑडियो परिचय पसंद करते हैं, तो हमारी टीम के सदस्य राचेल ने पोर्टो में [ओपन साइंस फेयर 2019](https://www.opensciencefair.eu/) में प्रस्तुत किया और उसका डेमो [ओरियन पॉडकास्ट](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/) द्वारा रिकॉर्ड किया गया था। _द ट्यूरिंग वे_ सिंहावलोकन मिनट 5:13 से शुरू होता है।
```

semantically :
यदि आप परियोजना के लिए एक ऑडियो परिचय sunna chahte hain, ....

spelling:
रेचल,

पोर्तो shahar   (add the shahar to Porto to make it clear while reading that it's a city)

--
2019 में प्रस्तुत किया --> semantically would become --> 2019 me apni prastuti rakhi aur….

---

ओरियन पॉडकास्ट --> See how pronounced- if pronounced – o-ri-yon, then the one mentioned is correct. 
If pronounced ‘o-ra-yen’ then - ओरयेन